### PR TITLE
#85 dispatch event when click on save

### PR DESCRIPTION
**Pull request by fkroockmann**
The event "on:save:click" must be dispatched into contribution application.
All application who need to save something must subscribe to this event.

### CODE REVIEW

File: src/tb/apps/contribution/views/contribution.view.index.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ define(
             bindUiEvents: function () {
                 jQuery(this.el).on('click', 'ul#edit-tab li', this.manageMenu);
                 jQuery(this.el).on('click', '#new-page', this.showNewPage);
+                jQuery(this.el).on('click', '#global-save', this.manageSave);
```
Comment (mickaelandrieu) [inline src/tb/apps/contribution/views/contribution.view.index.js]: can use create a variable instead ?
```js
var el = jQuery(this.el);
el.on('click', 'ul#edit-tab li', this.manageMenu)
    .on('click', '#new-page', this.showNewPage)
    .on('click', #global-save', this.manageSave)
;
```
should work.

File: src/tb/apps/main/controllers/main.controller.js [hunk 2]
```diff
@@ -32,6 +32,13 @@ define(['tb.core'], function (bbCore) {
 
         indexAction: function () {
             console.log('main index');
```
Comment (mickaelandrieu) [inline src/tb/apps/main/controllers/main.controller.js]: throw 'Non implemented for now'

or

call the logger

File: src/tb/apps/main/main.js [hunk 3]
```diff
@@ -19,7 +19,7 @@
 require.config({
     paths: {
         'main.routes': 'src/tb/apps/main/routes',
-        'main.controller': 'src/tb/apps/main/controllers/main.controller',
+        'main.main.controller': 'src/tb/apps/main/controllers/main.controller',
```
Comment (mickaelandrieu) [inline src/tb/apps/main/main.js]: nice catch ;)

**Comment by ndufreche** [comment]
:+1:

**Comment by mickaelandrieu** [comment]
Dont' merge before squash commits! reverted in https://github.com/backbee/BbCoreJs/pull/96, I will rewrite history.

**Comment by mickaelandrieu** [comment]
@fkroockmann can you re-push your pull request please ?

**Pull request merged**